### PR TITLE
Feat/content item inverse

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2212

Adds the inverse theme for ContentItem

### CODE REVIEW

File: packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js [hunk 1]
```diff
@@ -4,7 +4,9 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
+import { boolean } from '@storybook/addon-knobs';
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js:7]: @andysherman2121  Remove this line

File: packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js [hunk 2]
```diff
@@ -68,22 +85,26 @@ const ContentItem = ({ cta, copy, heading, mediaType, mediaData }) => (
  * @returns {*} media component
  */
 const _renderMedia = (type, data) => {
+  const inverse = boolean('inverse', false);
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js:88]: Shouldn't be using the storybook boolean in this.

File: packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js [hunk 3]
```diff
@@ -20,44 +22,59 @@ const { prefix } = settings;
...
+  return (
+    <div className={classnames} data-autoid={`${stablePrefix}--content-item`}>
+      {heading && (
+        <h4
+          data-autoid={`${stablePrefix}--content-item__heading`}
+          className={`${prefix}--content-item__heading`}>
+          {heading}
+        </h4>
+      )}
+      {_renderMedia(mediaType, mediaData)}
```
Comment (jeffchew) [inline packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js:57]: pass `inverse` into here as another argument

Comment (andysherman2121) [reply packages/react/src/patterns/sub-patterns/ContentItem/ContentItem.js]: Updated

**Pull request merged**